### DISCUSSION TITLE
zh_core_web_lg static embedding come from where?

### DISCUSSION THREAD

**Discussion by rgib37190** [explosion/spaCy#8184] [Language Support]
I want to know zh_core_web_lg static embedding come from where? Glove or fasttext? Thanks!!!

**Comment by polm** [comment]
I believe the current vectors are the same as in v2, and are trained on Wikipedia and OSCAR (Common Crawl) using fasttext. 

https://v2.spacy.io/models/zh#zh_core_web_lg

This info got omitted when building the 3.0 models, but we'll be putting it back for 3.1.

**Reply by rgib37190** [reply]
Thanks for reply!!! But I see it is trained on OntoNotes.  Which we use?

**Reply by polm** [reply]
The models (Parser, tagger, NER) are trained on OntoNotes. The vectors are trained separately and do not use OntoNotes.

**Reply by rgib37190** [reply]
I got it. Thanks for your reply!!!
